scripts/deploy.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -e # exit with nonzero exit code if anything fails
+
+if [[ $TRAVIS_BRANCH == "master" && $TRAVIS_PULL_REQUEST == "false" ]]; then
+
+echo "Starting to update gh-pages\n"
+
+#copy data we're interested in to other place
+cp -R dist $HOME/gh-pages
```

Comment:
The `cp` origin is now `dist` but should be `gh-pages` as that's the `outputFile`'s directory in `scripts/build-index.js`. This needs to be the same.